### DISCUSSION TITLE
Stop joing hpool

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#4833] [General]
Well, I thought that people would be smart and wouldnt join a hacked/malicious pool somewhere in china. I guess I was wrong. The network now is dominated by a single entity and there is no point in bothering at all with chia right now. And morons continue to join hpool, installing malicious software on their pc etc... :(

**Comment by oytp** [comment]
Well.. It's better to earn something instead of waiting for 2 XCH for years, right? I hope you got the idea

**Reply by GeorgePapageorgakis** [reply]
Is hpool code opensource?

**Reply by oytp** [reply]
> Is hpool code opensource?

Probably yes, they have github branch too

**Reply by andrejreimer** [reply]
No its closed.

**Comment by moshenskyDV** [comment]
Sorry bro. I tried to catch reward for a three weeks. And my chances are getting lower and lower everyday. I decided to switch to hpool when I joined to 0xCH club (on reddit, it means 100 plots without reward) :( 
Sad stuff, w8 for a official protocol...

**Reply by GeorgePapageorgakis** [reply]
You ll get locked in there but its your choice too...

**Comment by cccat6** [comment]
I can understand you but people who have a small capacity have no choice but to join HPool. I wish HPool net space will drop one official pooling protocol out.
I know HPool did not do anything wrong. But I just don't want it to have about half of space.

**Comment by TUstudents** [comment]
You get the idea, that earning something small is a better approach than earning nothing at all?

**Reply by loki77m** [reply]
are any offical pool going to star?

**Reply by oytp** [reply]
> are any offical pool going to star?

There will not be any official pools dude. Only pools which use chia pool API, which isn't ready yet
